app/controllers/api/user_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::UserController < ActionController::Base
   include GDS::SSO::ControllerMethods
 
-  skip_before_filter :verify_authenticity_token
+  skip_before_filter :verify_authenticity_token, raise: false
```

Comment:
This is skipping a before filter, so surely it'd never raise / throw / return false anyway?